spring-integration-debezium/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" />
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.junit" level="warn" />
+		<Logger name="example" level="warn" />
+		<Logger name="org.testcontainers" level="INFO"/>
+		<Logger name="com.github.dockerjava" level="WARN"/>
+		<Logger name="org.springframework.integration.debezium" level="info"/>
+
+		<Logger name="org.junit.vintage.engine.discovery" level="error" />
```

Comment:
I think we need to revise these categories, since I believe some of them just copy-n-paste from somewhere else.